src/components/tiptap/floatingMenu/FloatingMenu.tsx

```diff
@@ -15,220 +13,142 @@ import {
   TableIcon,
   EmbedIcon,
 } from '@/icons'
-
-import { TiptapEditorUtils } from '@/utils/tiptapEditorUtils'
-import { ImagePickerUtils } from '@/utils/imagePickerUtils'
 import { useAppState } from '@/hooks/useAppState'
 
-interface IFloatingMenuContainer {
-  editor: Editor
+const FloatingContainerBtn = ({
+  handleClick,
+  label,
+  focus,
+}: {
+  handleClick: () => void
+  label: string
+  focus: boolean
+}) => {
+  return (
+    <button
+      className={`flex flex-row gap-x-2.5 items-center py-1.5 px-3 cursor-pointer outline-none ${
+        focus && 'bg-new-white-2'
+      } display-block`}
+      onClick={() => {
+        handleClick()
+      }}
+    >
+      <div>
+        {label === 'Heading 1' ? (
+          <H1Icon />
+        ) : label === 'Heading 2' ? (
+          <H2Icon />
+        ) : label === 'Heading 3' ? (
+          <H3Icon />
+        ) : label === 'Text' ? (
+          <TextIcon />
+        ) : label === 'Autofill Fields' ? (
+          <AutofillIcon />
+        ) : label === 'Bullet List' ? (
+          <BulletListIcon />
+        ) : label === 'Numbered List' ? (
+          <NumberedListIcon />
+        ) : label === 'Upload' ? (
+          <UploadIcon2 />
+        ) : label === 'Embed' ? (
+          <EmbedIcon />
+        ) : label === 'Link' ? (
+          <LinkIcon />
+        ) : label === 'Table' ? (
+          <TableIcon />
+        ) : label === 'Callout' ? (
+          <CalloutIcon />
+        ) : (
+          <></>
+        )}
```

Comment:
@aatbip wouldn't `switch` be more appropriate here, even better if we could create a map of `label` and corresponding `component`, eg `{ 'Heading 1': H1Icon, 'Heading 2': H2Icon... }`